### ISSUE TITLE
Implement gen_conformers

### ISSUE THREAD

**Issue by Ben-Payton** [Ben-Payton/get_molecule_xyz#4]
gen_conformers is a function that will take some xyz file which is deemed to be inadequate and generate some number of conformers as sample geometries. Is this an accurate description?

**Comment by spackman** [comment]
yes, this seems accurate. the function should take the geometry as input (either from a file or a string), and the number of conformers selected. the function may also take the name for an output file or directory to place the conformers in, or may use some default names
